### PR TITLE
DB WK3

### PR DESCRIPTION
**Pull request by Nadoshka**

### CODE REVIEW

File: Week3/app.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  addReminder,
+  updateItem,
+  deleteItem,
+  deleteList,
+} = require('./mysql');
+
+app.use(bodyParser.json());
+
+app.post('/users/:id', (req, res) => {
+  addUser(req.body.userName, req.params.userId)
```
Comment (gajduk) [inline Week3/app.js]: very nice and clean 👍

File: Week3/mysql.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+function addItem(ToDoListId, description) {
+  return createdDb.execQuery(`INSERT INTO Item(ToDoListId, description) VALUES (?, ?)`, [
+    ToDoListId,
+    description,
+  ]);
+}
+
+function addReminder(ToDoListId, reminder) {
+  return createdDb.execQuery(`INSERT INTO reminder(ToDoListId, reminder) VALUES (?, ?)`, [
```
Comment (gajduk) [inline Week3/mysql.js]: prepared statements 👍

File: Week3/app.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    .then(data => res.status(200).json(data))
+    .catch(error => res.status(500).send({ message: "Item couldn't be deleted!", error }));
+});
+
+app.delete('/ToDoList', (req, res) => {
+  deleteList(req.params.ToDoListId, res)
+    .then(data => res.status(200).json(data))
+    .catch(error => res.status(500).send({ message: 'Item not found!', error }));
+});
+
```
Comment (gajduk) [inline Week3/app.js]: I dont see any way to read data from this app ??  Did you forget to add some get methods?

File: Week3/app.js (hunk 1 above)
Comment (Nadoshka) [reply Week3/app.js]: Thanks a lot for your feedback!

**Pull request closed without merging**